PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,35 +1,21 @@
 ## Description
 
-*(Instructions: replace text in this and all sections with your own text)*
+Provide a detailed description of what this PR does. What problem does it fix? What new capability does it add?
 
-Provide a detailed description of what this PR does.
+## Issue(s) addressed
 
-What problem does it fix? What new capability does it add?
-
-[See the JEDI Documentation](https://jointcenterforsatellitedataassimilation-jedi-docs.readthedocs-hosted.com/en/latest/inside/practices/pullrequest.html) for further information.
-
-### Issue(s) addressed
-
-Link the issues to be closed with this PR
-- fixes #<issue_number>
-
-If this does not close an existing issue, then [be sure to add a Pipeline, Label, Estimate, Assignees, and Epic](https://jointcenterforsatellitedataassimilation-jedi-docs.readthedocs-hosted.com/en/latest/inside/practices/issues.html)
-
-## Acceptance Criteria (Definition of Done)
-
-What does it mean for this PR to be finished?
+Resolves #<issue_number>
 
 ## Dependencies
 
-If there are PRs that need to be merged before or along with this one, please add "waiting for another PR" label and list the dependencies in the other repositories (example below). Note that the branches in the other repositories should have matching names for the automated tests to pass.
-Waiting on the following PRs:
-- [ ] waiting on JCSDA/eckit/pull/<pr_number>
-- [ ] waiting on JCSDA/atlas/pull/<pr_number>
+List the other PRs that this PR is dependent on:
```

Comment:
In general I like the concise instructions. However, I've gotten tripped up with some subtleties beyond just knowing that this PR goes with another PR (or another set of PRs). For example, if you have a ioda and idoa-data PR set and you happen to merge the ioda PR first, the tests that get kicked off on the new develop branch will use the ioda-data develop branch before the associated ioda-data changes and ctests will fail. Then the test failure shows up on displays such as the status bars in the README.md file and the little checkmark that shows up in github when you show the develop branch (in the "Code" tab. In this case it would be helpful to note that the idoa-data PR needs to be merged before the ioda PR.

I think it might be useful in the Dependencies section to mark the kind of dependency as noted above. Perhaps something like this in the ioda PR
- [ ] merge with ioda-data PR
    merge order: ioda-data, ioda

And for the ioda-data PR
- [ ] merge with ioda PR
    merge order: ioda-data, ioda

Would it make sense to add instructions to specify the merge order in this template?

Reply:
I think that the merge order is important to know for repo maintainers (perhaps we can create separate instructions for this to make it clear what the order should be?), but not essential to know or specify for anyone who opens a PR. What do you and others think?

Reply:
I agree with @shlyaeva, people submitting PRs don't necessarily know this. The template also needs to be independent of the repositories. And the `build group` syntax that will be added as part of the CI pipeline work will cover this aspect to some extend.

Reply:
Good point @shlyaeva. I agree that this information is for repo owners. I'm good with the template as is, and thanks for putting this together.